packages/backend-test-utils/src/database/TestDatabases.ts

```diff
@@ -113,7 +115,7 @@ export class TestDatabases {
    * Returns a fresh, unique, empty logical database on an instance of the
    * given database ID platform.
    *
-   * @param id The ID of the database platform to use, e.g. 'POSTGRES_13'
+   * @param id - The ID of the database platform to use, e.g. 'POSTGRES_13'
```

Comment:
Not a fan of the dashes honestly, is this a preferred format?

Reply:
It's almost like it takes the old pre-TS format where you were supposed to write a type there in curly braces but then leaving a dash instead kinda unnecessarily

Reply:
TSDoc says there has to be one, and api-extractor produces warnings if they're not there, so figured we might as well have them and not make a fuss 😅. I do remember having them in JSDoc too.

https://tsdoc.org/pages/tags/param/